README.md

```diff
@@ -1,20 +1,25 @@
 # Monk [![Build Status](https://travis-ci.org/brenoalvs/monk.svg?branch=master)](https://travis-ci.org/brenoalvs/monk)
+<div align="center">
```

Comment:
Remove `<div align="center">` and just use the bigger banner image here `1544x500`